source/standards/programming-languages.html.md.erb

```diff
@@ -98,6 +98,24 @@ We used Scala in the early days of GDS. GOV.UK Licensing is the only remaining
 application written in Scala but we’ve found it hard to support because of a lack
 of skills in GDS. Do not use Scala for new projects.
 
+## Mobile Development
+
+For developing mobile apps, we use Swift for iOS and Kotlin for Android.
+
+These languages have been chosen over using frameworks such as Flutter or .NET MAUI so that the apps
+feel like a native Android/iOS app without writing extra code to account for the specific OS in Flutter, for example. 
```

Comment:
I think this is a great statement, and one I might like to see expanded.

I wonder if one of the mobile seniors or folks involved with these decisions wouldn't mind chipping in here?

I think there's a candidate here for a separate GDS Way page about "preferences for native first / web first or hybrid". 

It'd be good if we could write this paragraph in a way we could link off to that when we have more guidance later.

Reply:
I think we'd be overreaching here to specify in this that we have an organisational preference for single platform mobile apps over cross platform tooling - as I understand DI and GOV.UK both made the decision to go single platform based on individual project requirements.

I think it'd be better to just leave how someone would approach a cross platform app as undefined here as we've not yet established a pattern/preferred framework in the org - and maybe we never will? And then to amend the above sentence to be: "For developing native mobile apps, we use Swift for iOS and Kotlin for Android."

I imagine there could be a sentence or two to explain why Kotlin over an alternative like Java for native Android development and why Swift over Objective-C for native iOS - but it could be a little superfluous given how dominant both languages in the sphere. So as an alternative something that just says "These languages have been chosen as they are the common industry standard and vendor recommended languages to use for their respective platforms"